besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1374,7 +1374,15 @@ private PrivacyParameters privacyParameters() throws IOException {
       privacyParametersBuilder.setEnabled(true);
       privacyParametersBuilder.setEnclaveUrl(privacyUrl);
       if (privacyPublicKeyFile() != null) {
-        privacyParametersBuilder.setEnclavePublicKeyUsingFile(privacyPublicKeyFile());
+        try {
+          privacyParametersBuilder.setEnclavePublicKeyUsingFile(privacyPublicKeyFile());
+        } catch (final IOException e) {
+          throw new ParameterException(
+              commandLine, "Problem with privacy-public-key-file: " + e.getMessage(), e);
```

Comment:
We probably don't need to print the msg with e.getMessage(). It will be printed by the logger since we are passing the exception as a parameter.